releases/protocol/release-specs-v1.7.14-Interim.md

```diff
@@ -0,0 +1,124 @@
+
+### Impact:
+* There is a new enable epoch definition for this feature, called `DynamicESDTEnableEpoch`
+* There are changes to the configuration options in the gas schedule files, new values were added for `ESDTModifyRoyalties`, `ESDTModifyCreator`,`ESDTNFTRecreate`,`ESDTNFTUpdate`, `ESDTNFTSetNewURIs`
+* No binary flags changes
+* API endpoints `/:address/nft/:tokenIdentifier/nonce/:nonce` and `/:address/esdt` will now return token type as well
+
+### Relevant PRs:
+- [#5806](https://github.com/multiversx/mx-chain-go/pull/5806) - ESDT improvements integration
+- [#5682](https://github.com/multiversx/mx-chain-go/pull/5682) - ESDT v2 implementations on SystemSC - dynamic tokens and tokenType
+- [#6284](https://github.com/multiversx/mx-chain-go/pull/6284) - ESDT Improvements integration tests
+- [#6194](https://github.com/multiversx/mx-chain-go/pull/6194) - ESDT improvements chain simulator tests and fixes
+- [#6262](https://github.com/multiversx/mx-chain-go/pull/6262) - ESDT improvements integration tests part2 + fixes
+- [#6250](https://github.com/multiversx/mx-chain-go/pull/6250) - Do not allow NFTs to be upgraded to dynamic
+- [#6263](https://github.com/multiversx/mx-chain-go/pull/6263) - Add more testing scenarios
+- [#6242](https://github.com/multiversx/mx-chain-go/pull/6242) - Add esdt token type api
+- [#6220](https://github.com/multiversx/mx-chain-go/pull/6220) - Token type in altered accounts
+- [#6350](https://github.com/multiversx/mx-chain-go/pull/6350) - Fix missing role + Use setSpecialRole function in tests
+- [#6357](https://github.com/multiversx/mx-chain-go/pull/6357) - Dynamic roles singular
+
+## 3. Crypto API, new Opcodes and EGLD in MultiESDTTransfer [#6139](https://github.com/multiversx/mx-chain-go/pull/6139)
+
+With this feature, new opcodes were enabled for developers, among with new crypto endpoints(`VerifySecp256r1`, `VerifyBLSSignatureShare`, `VerifyBLSMultiSig`).
+Another improvement was EGLD as part of MultiESDTTransfer (EGLD-000000).
+
+### Impact:
+* There are new enable epochs definitions for this feature, called `EGLDInMultiTransferEnableEpoch` and `CryptoOpcodesV2EnableEpoch`
+* There are changes to the configuration options in the gas schedule files, new values were added for `VerifySecp256r1`, `VerifyBLSSignatureShare`,`VerifyBLSMultiSig`
+* No binary flags changes
+* No API endpoints changes
+
+### Relevant PRs:
+- [#6334](https://github.com/multiversx/mx-chain-go/pull/6334) - Multi transfer execute by user flag
+- [#6313](https://github.com/multiversx/mx-chain-go/pull/6313) - Added egld with multi transfer test scenario
+
+## 4. Refactor persister factory [#6001](https://github.com/multiversx/mx-chain-go/pull/6001)
+
+The storage unit package was refactored, adding the option of using static and non-static storers.
+
+### Impact:
+* No activation epoch
+* No configuration files changes
+* No binary flags changes
+* No API endpoints changes
```

Comment:
Any changes within the database structure that is worth mentioning?

Alternatively, replace section with a simple statement e.g. no impact on validators / developers etc.?